docs/tutorials/quick-start-model.md

```diff
@@ -23,20 +23,20 @@ In the **dial-docker-compose/model/core** folder, you can find a [config.json](h
 In `config.json`, you can add your Azure model credentials to the chat configuration:
 
 * Supply your **Azure API Keys** for your deployments for the `key` parameter.
-* Replace `https://AZURE_DEPLOYMENT_URL` with your GPT **endpoint** for the `endpoint` parameter. **Note**: in the endpoint, replace `gpt-4` with your Azure deployment name, in case it is different.
+* Replace `http://azure_deployment_url` with your GPT **endpoint** for the `endpoint` parameter. **Note**: in the endpoint, replace `gpt-4` with your Azure deployment name, in case it is different.
 
   ```json
         "upstreams": [
         {
-          "endpoint": "https://AZURE_DEPLOYMENT_URL/openai/deployments/gpt-4/chat/completions",
+          "endpoint": "http://azure_deployment_url/openai/deployments/gpt-4/chat/completions",
```

Comment:
Is it worth renaming azure_deployment_url to azure_deployment_host or something similar?

Reply:
renamed to `azure_deployment_host`